CRM/Civicaseextras/APIWrappers/Case/Getcaselist.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class CRM_Civicaseextras_APIWrappers_Case_Getcaselist implements API_Wrapper {
+    /**
+     * Implements API_Wrapper::fromApiInput().
+     *
+     * Alters the Case Getcaselist action so it includes the case duration custom field to the response.
+     *
+     * @param array $apiRequest
+     * @return array
+     */
+    public function fromApiInput($apiRequest) {
+      $return = CRM_Utils_Array::value('return', $apiRequest['params'], []);
+      $caseDuration = _civicaseextras_get_caseDurationField();
+
+      if ($caseDuration) {
+        $return[] = 'custom_' . $caseDuration['id'];
```

Comment:
it would be extremely rare for it to be in the return array because you need to find out the custom id field first and you can't do that from the FE unless you provide it from the BE.

This is exactly we are returning this field by default, so the FE doesn't need to request the custom field ID and then do another request to include the field.